DcCore/DcCore/DC/events.swift

```diff
@@ -167,6 +164,11 @@ public class DcEventHandler {
                 NotificationCenter.default.post(name: eventConnectivityChanged, object: nil)
             }
 
+        case DC_EVENT_ACCOUNTS_BACKGROUND_FETCH_DONE:
```

Comment:
According to [the docs](https://c.delta.chat/classdc__accounts__t.html#a23e7f5095f52a80a605e0266f686f3b7), the [`DC_EVENT_CONNECTIVITY_CHANGED`](https://c.delta.chat/group__DC__EVENT.html#ga3ced1bbde752feba46ab089611287d71) gets emitted when work is done (but also when connectivity changes, so this sounds like a little ... workaround).

From what I understand now, we don't use the `dc_accounts_all_work_done` anymore, but replaced it with all new [`dc_accounts_background_fetch`](https://c.delta.chat/classdc__accounts__t.html#a4b735a1e51222a06aa992773e42d06b0). Would it make sense to remove `DcAccounts.isAllWorkDone()`?

Also, if I understand this correctly, as a consequence of this change, we don't wait anymore for things to update when the connectivity changes. Instead, we wait for the work to be finished as part of the iOS-background fetch, right?